jolt-core/src/jolt/instruction/div.rs

```diff
@@ -190,7 +201,26 @@ impl<const WORD_SIZE: usize> VirtualInstructionSequence for DIVInstruction<WORD_
             advice_value: None,
         });
 
-        virtual_sequence
+        virtual_trace.push(RVTraceRow {
```

Comment:
Previously, we had elided the MOVE instruction as described in the Jolt paper as an optimization, instead writing the advice directly to the destination register
But as it turns out, we actually did need the separate virtual register + MOVE for the edge case where `rd` is equal to `rs1` or `rs2`